internal/tiltfile/extension/github_fetcher.go

```diff
@@ -0,0 +1,49 @@
+package extension
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"time"
+)
+
+type GithubFetcher struct {
+}
+
+func NewGithubFetcher() *GithubFetcher {
+	return &GithubFetcher{}
+}
+
+const githubTemplate = "https://raw.githubusercontent.com/windmilleng/tilt-extensions/master/%s/Tiltfile"
+
+func (f *GithubFetcher) Fetch(ctx context.Context, moduleName string) (ModuleContents, error) {
```

Comment:
I decided not to write tests for this because I didn't feel like it would be useful. Happy to take pushback, though.

Reply:
👍